platformcomponents/desktop/messages.json

```diff
@@ -72,6 +72,36 @@
           "text": "@theme-text-warning-normal"
         }
       }
+    },
+    "badge": {
```

Comment:
We should have a separate component for this, to follow the structure in the Figma file. Please open a new json file under platformcomponents/desktop